Configuration/Eras/python/Era_Run2_2017_trackingLowPU_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
+
+Run2_2017_trackingLowPU = cms.ModifierChain(Run2_2017, trackingLowPU)
```

Comment:
@makortel if you can do the cleanup using `copyAndExclude`, including the fix I mentioned in #15496 for the 2018 Era, that would be great